cql3/expr/expression.cc

```diff
@@ -1087,7 +1088,7 @@ std::ostream& operator<<(std::ostream& os, const column_value& cv) {
 std::ostream& operator<<(std::ostream& os, const expression& expr) {
     expression::printer pr {
         .expr_to_print = expr,
-        .mark_unresolved_identifiers = true
+        .debug_mode = true
```

Comment:
please conflate this patch with previous patch, which added this flag with a different name